R/recordedplot.r

```diff
@@ -27,6 +27,7 @@ repr_png.recordedplot <- function(p,
 	#special
 	res       = getOption('repr.plot.res'),
 ...) {
+	if (!capabilities('png')) { return NULL }
```

Comment:
yeah, although i can’t imagine what setup would result in this being false… embedded R? :laughing:

yeah, you can configure it to build without but i doubt anyone using repr would ever do this. still, why not put the check in?